src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -65,6 +65,7 @@ def __init__(self):
         self.mib_oid_list = []
 
         self.queue_type_map = {}
+        self.if_id_namespace = {}
```

Comment:
>if_ [](start = 13, length = 3)

The concept in this MIB is port, not interface. Suggest rename `if_*` to `port_*`

Reply:
I have modified the naming in this file.